SOP/SOP-push.adoc

```diff
@@ -32,9 +32,7 @@ All alerts should appears as green in the Prometheus Alert Monitoring.
 
 == Prometheus Alerts Procedures
 
-The following guide outlines the steps required to solve issues which will be alert by the Prometheus alerts configured to monitoring the  https://github.com/aerogear/aerogear-unifiedpush-server[UnifiedPush Server] when it is managed, installed and configured via the https://github.com/aerogear/unifiedpush-operator[UnifiedPush Operator].
```

Comment:
I do not think that it should be removed. 
It brings context and explains what the goal of it.

Reply:
This line is duplicated above.